src/traces/box/attributes.js

```diff
@@ -202,5 +213,16 @@ module.exports = {
         },
         editType: 'plot'
     },
-    fillcolor: scatterAttrs.fillcolor
+    fillcolor: scatterAttrs.fillcolor,
+    hoveron: {
+        valType: 'flaglist',
+        flags: ['boxes', 'points'],
+        dflt: 'boxes',
+        role: 'info',
+        editType: 'style',
+        description: [
+            'Do the hover effects highlight individual boxes ',
+            'or jitter points or both?'
```

Comment:
I wouldn't call them jitter points - sample points?

Reply:
Good eye. Thanks!

Reply:
done in https://github.com/plotly/plotly.js/pull/2094/commits/a1ea1e83717c99aa8cb0e64a1fa493cae933cc64